user-guide/lobby-rules.md

```diff
@@ -104,3 +104,32 @@ If a user has been found to directly target a person in a server, they will be r
 Remember our Golden Rule is to treat others the same way you would want to be treated. TripleA 
 is one of the greatest Axis and Allies games and community. We want everyone to enjoy their
  time here, so please keep it clean and have fun!
+
```

Comment:
Why is the list of active moderators useful?
Is this just another thing for us to maintain? When we add moderators, or remove them, do we really want to *have* to update this list?

Reply:
Do we leave the Mods and Admins hidden?  I think it should be general knowledge to all users.  Just my two cents.

Reply:
The moderators are marked as such in the chatter list. In that sense it is not hidden. I'm not sure if having the list of moderators be here also solves any issues of a player being confused as to who is a moderator.